docroot/themes/custom/mass_theme/templates/paragraphs/paragraph--rich-text.html.twig

```diff
@@ -36,8 +36,8 @@
  * @ingroup themeable
  */
 #}
-{% embed "@organisms/by-author/rich-text.twig" %}
+{# {% embed "@organisms/by-author/rich-text.twig" %} #}
```

Comment:
Huh... I'm worried about how this change might ripple out to other content types.  This paragraph type is used pretty heavily, right?  Maybe we should do some visual regression testing to figure out the scope of the change.